### PR TITLE
get_nncontext -> init_nncontext and update docs for pip

### PR DESCRIPTION
**Pull request by hkvision**
Fix https://github.com/intel-analytics/zoo/issues/287

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Analytics Zoo provides a set of easy-to-use, high level pipeline APIs that nativ
    ```
```
Comment (jason-dai) [inline README.md]: Change to 
1. Initialize _NNContext_ and load images into DataFrames using `NNImageReader`

Comment (hkvision) [reply README.md]: updated.

**Comment by hkvision** [comment]
jenkins passed http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/564/

**Comment by hkvision** [comment]
app tests passed http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Python-AppTests/60/

**Pull request merged**